feature/gnmi/ate_tests/telemetry_basic_check_test/telemetry_basic_check_test.go

```diff
@@ -446,7 +446,7 @@ func TestComponentParent(t *testing.T) {
 func TestSoftwareVersion(t *testing.T) {
 	dut := ondatra.DUT(t, "dut")
 
-	cards := components.FindComponentsByType(t, dut, cpuType)
+	cards := components.FindComponentsByType(t, dut, supervisorType)
```

Comment:
this may not help as well, since supervisorType is a hardware component and according to oc model software version is only for software components:

leaf software-version {
      type string;
      description
        "For software components such as operating system or other
        software module, this is the version of the currently
        running software.";
    }

Reply:
It may be implied that SW component (NOS) is running on top of a supervisor module and its version is reported. And supervisor modules program linecard accordingly during a boot/upgrade procedure, using a specific SW version as well.

In this case, control module#1 can report a different sw version from control module#2 under some circumstances (e.g.  during an upgrade of a system with redundant control modules) and so on.

But I'll leave it up to openconfig experts to decide.

Reply:
@dplore can you help provide some clarify from OC? Thanks. 

The current README: https://github.com/openconfig/featureprofiles/blob/main/feature/gnmi/ate_tests/telemetry_basic_check_test/README.md

Reply:
Current README seems questionable, we will file a README PR to clarify things later.